source/dub/packagesupplier.d

```diff
@@ -322,7 +324,18 @@ package alias FallbackPackageSupplier = AutoImplement!(AbstractFallbackPackageSu
 private template fallback(T, alias func)
 {
 	enum fallback = q{
-		scope (failure) return m_fallback.%1$s(args);
+		import std.range : back, dropBackOne;
+		scope (failure)
+		{
+			foreach (m_fallback; m_fallbacks.dropBackOne)
+			{
+				try
+					return m_fallback.%1$s(args);
+				catch(Throwable)
+				    assert(1);
+			}
+			return m_fallbacks.back.%1$s(args);
```

Comment:
The recursive structure was setup to chain exceptions, so that users get a list of unreachable sites. Not too important though.